tsconfig.packages.json

```diff
@@ -0,0 +1,16 @@
+{
+  "compilerOptions": {
+    "composite": true,
+    "esModuleInterop": true,
+    "module": "CommonJS",
+    "moduleResolution": "node",
+    /* NOTE: This must be synchronized with the `moduleNameMapper` option in `jest.config.packages.json` */
+    "paths": {
```

Comment:
This setting allows one package to import another package in the same way whether developing or building for release, yet specify different behavior for how the import path should be resolved. That is, if `packages/ens-controller` imports `@metamask/base-controller`, then `@metamask/base-controller` will be used as the import path in the released version of `@metamask/ens-controller`; but when developing, `packages/ens-controller` will import `packages/base-controller`.